### PR TITLE
Issue #164 AsyncHBase compatibility with HBase 1.3.1

### PR DESCRIPTION
**Pull request by karanmehta93**
Following changes are made.
1. Client Protos upgraded from HBase 0.98 to 1.3
2. Makefiles modified for adding new files
3. Handling of 'more_results_in_region' bit in ScanResponse
If this bit is unset then new GetNextRowsRequest is not issued since the scanner will be already closed on server side and will result into exceptions on client side if the RPC is still made.
4. Added an integration test that scans across multiple regions, thus involving multiple open, close and getnextrows requests.

### CODE REVIEW

**Comment by karanmehta93** [comment]
I am also not sure as to why the build is failing.

**Comment by karanmehta93** [comment]
@manolama 
Can you review it?

File: test/TestIntegration.java [hunk 1]
```diff
@@ -351,6 +353,57 @@ public void appendTwiceBatchedReturnRead() throws Exception {
     assertEquals(write_time, kvts, 5000.0);  // Within five seconds.
   }
 
+  @Test
+  public void basicScanMultipleRegions() throws Exception {
+    splitTable(table, "s10");
+    splitTable(table, "s20");
+    splitTable(table, "s30");
+    List<Deferred<Object>> list = new ArrayList<>();
```
Comment (manolama) [inline test/TestIntegration.java]: The ``<>`` isn't allowed in Java 6. We're still bw compatible for existing users.

**Comment by manolama** [comment]
Drats, just tested. It works fine with 1.3.1 but breaks with previous versions (at least 1.0.0 and 1.1.2)

**Comment by karanmehta93** [comment]
Let me check it once again. I tested it with 0.98, 1.2, 1.3 and worked fine. Can you point to the location from where you got these versions and what errors you found? 
Also can you describe which other versions I should check with?

**Comment by manolama** [comment]
0.98 worked, lemme double check the 1.0.0 and 1.1.

**Comment by manolama** [comment]
1.1.2 is here: https://archive.apache.org/dist/hbase/1.1.2/
and here's the 1.0.0: https://archive.apache.org/dist/hbase/hbase-1.0.0/
Still playing around with em.

**Comment by manolama** [comment]
Never mind, I had some goof-up with different HBase versions pointing at the same files :) I'll fix the integration test and merge.

**Comment by manolama** [comment]
Merged in 80c6b86084a1bf75078e1c4419b25193117c116e. Thanks!

**Comment by karanmehta93** [comment]
Thank you!!

**Pull request closed without merging**